en/px-platform.json

```
@@ -479,6 +479,9 @@
               "title": "Congrats! Your PayPal account was successfully connected",
               "description": "Let’s take some paid research studies to celebrate."
             },
+            "verifiedMustVerify": {
+              "title": "Congrats! We’ve confirmed your PayPal account is verified and you’re  good to go"
```

Lil extra space snuck in here 🥷 

```suggestion
              "title": "Congrats! We’ve confirmed your PayPal account is verified and you’re good to go"
```

---

Nice catch! Thanks will get that sorted